build/configs/artik053/tc/defconfig

```diff
@@ -1224,15 +1200,31 @@ CONFIG_SYSIO_UTC_I2C=y
 CONFIG_SYSIO_UTC_PWM=y
 CONFIG_SYSIO_UTC_GPIO=y
 # CONFIG_EXAMPLES_TESTCASE_SYSTEMIO_ITC is not set
+CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC=y
+
+#
+# Specify your AP which you want to connect to
+#
+CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SSID="NULL"
+CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_PASSPHRASE="NULL"
```

Comment:
I intend to set this string NULL, because SSID and PASSPHARSE is not same for all.
But I think this is also seems not good because user should modify config.